hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2695,37 +2695,18 @@ def hpat_pandas_series_sort_values_impl(self, axis=0, ascending=True, inplace=Fa
             for i in self.isna():
                 if i:
                     na += 1
-            index = numpy.arange(len(self._data))
-            my_index = numpy.arange(len(self._data))
-            used_index = numpy.full((len(self._data)), -1)
+            indices = numpy.arange(len(self._data))
```

Comment:
Looks like `indices` is not required and `index_result` is enough. Correct me if I'm wrong.